cli/cmds/cluster/delete.go

```diff
@@ -29,6 +29,10 @@ func delete(clx *cli.Context) error {
 	if err != nil {
 		return err
 	}
+	namespace := "default"
```

Comment:
I think we can use pkg/controller/util package and add the defaultNamespace there as a constant like we do with the `K3kSystemNamespace`